dallinger/default_configs/local_config_defaults.txt

```diff
@@ -0,0 +1,35 @@
+[HIT Configuration]
+title = Stroop task
+description = Judge the color of a series of words.
+amt_keywords = Perception, Psychology
+lifetime = 24
+us_only = true
+approve_requirement = 95
+contact_email_on_error = youremail@gmail.com
+ad_group = Default psiTurk Stroop Example
+psiturk_keywords = stroop
+organization_name = New Great University
+browser_exclude_rule = MSIE, mobile, tablet
+
+[Database Parameters]
+database_url = sqlite:///participants.db
+table_name = turkdemo
+
+[Server Parameters]
+host = localhost
+port = 22362
+cutoff_time = 30
+logfile = server.log
+loglevel = 2
+debug = true
+login_username = examplename
+login_pw = examplepassword
+threads = auto
+
+[Task Parameters]
+experiment_code_version = 1.0
+num_conds = 1
+num_counters = 1
+
+[Shell Parameters]
+launch_in_sandbox_mode = true
```

Comment:
Yes, use the [Bartlett1932 config](https://github.com/Dallinger/Dallinger/blob/master/demos/bartlett1932/config.txt) instead:

```
[Experiment Configuration]
mode = sandbox
auto_recruit = true

[HIT Configuration]
title = War of the Ghosts
description = Read a brief story and answer some questions about it.
amt_keywords = Psychology, reading, text
base_payment = 1.00
lifetime = 24
duration = 0.1
us_only = true
approve_requirement = 95
contact_email_on_error = bartlett1932@mailinator.com
ad_group = War of the Ghosts
psiturk_keywords = ghosts
organization_name = Monsters University
browser_exclude_rule = MSIE, mobile, tablet

[Database Parameters]
database_url = postgresql://postgres@localhost/dallinger
table_name = psiturk
anonymize_data = false
database_size = standard-0

[Server Parameters]
dyno_type = free
num_dynos_web = 1
num_dynos_worker = 1
host = 0.0.0.0
notification_url = None
clock_on = false
logfile = -

[Shell Parameters]
launch_in_sandbox_mode = true
```